great_expectations/checkpoint/checkpoint.py

```diff
@@ -1,9 +1,8 @@
-import copy
```

Comment:
Minor: my stylistic preference would be to `import copy` and then use `copy.deepcopy()`.  Right now, this module uses both (which detracts from code readability).  Thank you! /cc @roblim